File: week4/natjecanje.py
def readData():
	nsr = raw_input()
	nsr = nsr.split()
	arrS = raw_input()
	arrR = raw_input()
	arrS = arrS.split()
	arrR = arrR.split()
	for i in range(0,len(arrS)):
		arrS[i] = int(arrS[i])
	for i in range(0,len(arrR)):
		arrR[i] = int(arrR[i])
	result = arrS[:];
	for i in arrS:
		# print i
		if (i-1) in arrR:
			arrR.remove(i-1)
			result.remove(i)
		elif (i+1) in arrR:
			arrR.remove(i+1)
			result.remove(i)
	print(len(result))
readData()

File: week4/Natjecanje.cpp
#include <stdio.h>
#include <math.h>
#include <stdlib.h>
#include <iostream>
#include <string.h>

using namespace std;

void readData(){
	int n, s, r;
	scanf("%d%d%d",&n,&s, &r);
	int arrS[s], arrR[r];
	for (int i = 0; i < s; ++i)
	{
		scanf("%d",&arrS[i]);
		printf("%d",arrS[i]);
	}
	printf("\n");
	for (int i = 0; i < r; ++i)
	{
		scanf("%d",&arrR[i]);
		printf("%d",arrR[i]);
	}
}
int main(int argc, char const *argv[])
{
	readData();
	return 0;
}
